lib/runtime/wasmer/exports.go

```diff
@@ -175,7 +175,47 @@ func (in *Instance) PaymentQueryInfo(ext []byte) (*types.TransactionPaymentQuery
 		return nil, err
 	}
 
-	i := new(types.TransactionPaymentQueryInfo)
+	i := new(types.RuntimeDispatchInfo)
+	if err = scale.Unmarshal(resBytes, i); err != nil {
+		return nil, err
+	}
+
+	return i, nil
+}
+
+// QueryCallInfo returns information of a given extrinsic
+func (in *Instance) QueryCallInfo(ext []byte) (*types.RuntimeDispatchInfo, error) {
+	encLen, err := scale.Marshal(uint32(len(ext)))
+	if err != nil {
+		return nil, err
+	}
+
+	resBytes, err := in.Exec(runtime.TransactionPaymentCallAPIQueryCallInfo, append(ext, encLen...))
+	if err != nil {
+		return nil, err
+	}
+
+	i := new(types.RuntimeDispatchInfo)
```

Comment:
`is` does not looks like a clear name, so I would suggest to change to `dispatchInfo`

```suggestion
	dispatchInfo := new(types.RuntimeDispatchInfo)
```